server/src/main/java/org/candlepin/model/Owner.java

```diff
@@ -493,7 +493,13 @@ public void setContentAccessMode(String contentAccessMode) {
      * @return String the value
      */
     public String getContentAccessModeList() {
-        return this.contentAccessModeList;
+        String[] modeList = contentAccessModeList == null ? new String[0] : contentAccessModeList.split(",");
+        String ammend = "";
+        if (!ArrayUtils.contains(modeList, ContentAccessCertServiceAdapter.DEFAULT_CONTENT_ACCESS_MODE)) {
```

Comment:
@wottop sounds like there might need to be a larger discussion about whether empty should really be a valid option.